res/controllers/Yaeltex MiniMixxx.midi.xml

```diff
@@ -0,0 +1,446 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MixxxMIDIPreset mixxxVersion="2.3.0" schemaVersion="1">
+  <info>
+    <name>Yaeltex MiniMixxx</name>
+    <author>Owen Williams</author>
+    <description>MIDI Mapping for the Yaeltex MiniMixx</description>
```

Comment:
Officially sanctioned by the Redundancy Department of Redundancy. 😉

The description be informative and not just repeat the name, e.g.:

```suggestion
    <description>A compact DJ controller specifically designed for Mixxx.</description>
```